dask/dataframe/core.py

```diff
@@ -415,6 +415,106 @@ def map_partitions(self, func, *args, **kwargs):
         """
         return map_partitions(func, self, *args, **kwargs)
 
+    @insert_meta_param_description(pad=12)
+    def reduction(self, chunk, aggregate=None, meta=no_default,
+                  token=None, chunk_kwargs=None, aggregate_kwargs=None,
+                  **kwargs):
+        """Generic row-wise reductions.
+
+        Parameters
+        ----------
+        chunk : callable
+            Function to operate on each partition. Should return a
+            ``pandas.DataFrame``, ``pandas.Series``, or a scalar.
+        aggregate : callable, optional
+            Function to operate on the concatenated result of ``chunk``. If not
+            specified, defaults to ``chunk``.
+
+            The input to ``aggregate`` depends on the output of ``chunk``.
+            If the output of ``chunk`` is a:
+            - scalar: Input is a Series, with one row per partition.
+            - Series: Input is a DataFrame, with one row per partition. Columns
+              are the rows in the output series.
```

Comment:
Ah, never mind, I now see that this is is aligned with how dataframes handle individual rows.